PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -1085,28 +1085,33 @@ void AliEmcalJetTask::SetTrackingEfficiencyFromYAML(std::string period , std::st
   std::vector <Double_t> trackingUncertainty;
   bool res = fYAMLConfig.GetProperty("ptBinning", ptBinning, false);
   Int_t nPtBins = ptBinning.size()-1;
-  double* aptBinning = &ptBinning[0];
-
-  TString sPeriod(period);
+  double* aptBinning = ptBinning.data();
+
+  TObjArray *sPeriodCentArray = TString(periodCent).Tokenize("_");
+  TString sPeriod = sPeriodCentArray->At(0)->GetName();
+  TString sCent = "";
+  Bool_t isCentrality = kFALSE;
+  if(sPeriodCentArray->GetEntries()==3) {
+    sCent = Form("%s_%s",sPeriodCentArray->At(1)->GetName(),sPeriodCentArray->At(2)->GetName());
```

Comment:
`Form` can buffer overflow. It's not common, but there's no need to take a risk - use `TString::Format(...)` instead